google-drive.yaml

```diff
@@ -0,0 +1,114 @@
+name: Google Drive
+description: |
+  ![Google Drive MCP Server](https://img.shields.io/badge/Google%20Drive-MCP%20Server-blue?style=for-the-badge&logo=google-drive)
+  ![Python](https://img.shields.io/badge/Python-3.13+-green?style=for-the-badge&logo=python)
+  ![FastMCP](https://img.shields.io/badge/FastMCP-2.11.1+-orange?style=for-the-badge)
+  ![License](https://img.shields.io/badge/License-MIT-yellow?style=for-the-badge)
+
+  A Model Context Protocol (MCP) server for Google Drive integration, providing comprehensive file and folder management capabilities through a modern API interface.
+
+  ## Features
+  - **OAuth2 Authentication**: Secure Google OAuth integration with automatic token refresh
+  - **File Operations**: List, get, copy, update, and delete files with advanced filtering and search
+  - **Folder Management**: Create and organize folder hierarchies in personal and shared drives
+  - **Permission Control**: Comprehensive permission management with role-based access and sharing
+  - **Ownership Management**: Transfer file ownership to users within the same domain
+  - **Advanced Search**: Filter files by MIME type, name, modification time, and folder location
+  - **Shared Drive Support**: Full management of shared drives including creation, deletion, and renaming
+  - **Granular Permissions**: Create, update, and delete permissions for users, groups, domains, and public access
+
+  ## What you'll need to connect
+
+  **No Setup Required**: Authentication is handled automatically through OAuth2 workflow
+
+metadata:
+  categories: Business
+  allow-multiple: "true"
+icon: https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/Google_Drive_icon_%282020%29.svg/330px-Google_Drive_icon_%282020%29.svg.png
```

Comment:
nit: don't think the background on this is transparent.

Reply:
updated.